tests/test_http_server.py

```diff
@@ -3,12 +3,12 @@
 import pytest
 
 
-# @pytest.mark.dockertest
-# @pytest.mark.httpservertest
-# def test_password_cluster_is_up(local_docker_cluster_passwd):
-#     cluster = local_docker_cluster_passwd
-#     cluster.check_server()
-#     assert cluster.is_up()  # Should be true for a Cluster object
+@pytest.mark.dockertest
```

Comment:
thoughts on having a single test that can be paramaterized across clusters? (presumably we'd want to check if the server is up not just on `local_docker_cluster_passwd` and `local_docker_cluster_public_key`?

Reply:
Absolutely, sounds good! This could probably go in as a separate small PR. I can quickly iterate on this tomorrow.

Reply:
Absolutely, thanks for catching this.